format_checker/main.py

```diff
@@ -5,7 +5,7 @@
 import errorhandler
 from tso_iso_checker import run_checks_tso_iso
 from tic_fic_checker import run_checks_tic_fic
-from pr_checker import run_checks_pr
+from pr_checker import run_checks_pr,run_checks_gr
```

Comment:
Missing space after comma; are we running some style checker on these files?

Reply:
fixed,
I haven't noticed any style checking functions/calls in the repo

Reply:
Can you add `flake8`, if familiar with Python? If not, it's not super needed, because these scripts are not maintained by many people.

Reply:
A workflow for flake8 is added, there are several style errors in existing python files, I'll resolve them and commit in this PR

Reply:
done